src/components/Stake/assets/wallet.svg

```diff
@@ -0,0 +1,15 @@
+<svg width="49" height="49" viewBox="0 0 49 49" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M34.8086 24.0064V21.8636C34.8086 21.0745 34.137 20.435 33.3086 20.435H16.8086C15.9813 20.435 15.3086 19.7943 15.3086 19.0064C15.3086 18.2186 15.9813 17.5778 16.8086 17.5778H33.3086C34.137 17.5778 34.8086 16.9382 34.8086 16.1493C34.8086 15.3603 34.137 14.7207 33.3086 14.7207H16.8086C14.3271 14.7207 12.3086 16.6432 12.3086 19.0064V20.435V31.8636C12.3086 33.4415 13.6518 34.7207 15.3086 34.7207H33.3086C34.137 34.7207 34.8086 34.0812 34.8086 33.2921V31.1493C35.637 31.1493 36.3086 30.5097 36.3086 29.7207V25.435C36.3086 24.646 35.637 24.0064 34.8086 24.0064ZM32.5586 29.0064C31.7302 29.0064 31.0586 28.3669 31.0586 27.5778C31.0586 26.7888 31.7302 26.1493 32.5586 26.1493C33.387 26.1493 34.0586 26.7888 34.0586 27.5778C34.0586 28.3669 33.387 29.0064 32.5586 29.0064Z" fill="url(#paint0_linear)"/>
```

Comment:
Something I noticed with these icons is that I think the cropping is a little off in our exports. The coins tend to render smaller than the wallet, might be worth checking them :)

![image](https://user-images.githubusercontent.com/11708259/92505473-0c61a580-f1fc-11ea-8667-de1ebdd76be3.png)